centrallix/tests/test_moneytype_03.c

```diff
@@ -0,0 +1,36 @@
+#include "obj.h"
+#include <assert.h>
+#include <stdio.h>
+
+long long
+test(char** name)
+{
+    *name = "moneytype_03 - objDataToMoney";
+    MoneyType test = {0};
+
+    /** String Case **/
+    char data_ptr[] = "$4.50";
+    assert(objDataToMoney(2,data_ptr,&test) == 0);
+    assert(test.Value == 45000);
+
+    /** Double Case **/
+    double testDouble = 5.5;
+    assert(objDataToMoney(3,&testDouble,&test) == 0);
+    assert(test.Value == 55000);
```

Comment:
Would it be possible for a double to have more significant digits in the fractional part than can be stored in a MoneyType? If so could you add a case for that?

A case for a negative double would also be great